src/pages/AboutTheCreators.jsx

```diff
@@ -1,9 +1,11 @@
-export default function AboutTheCreators(){
-    return(
-        <h1>
-            Learn About the Creators
-        </h1>
+import AboutCreator from '../component/creator/creator';
 
+const AboutTheCreators = () => {
```

Comment:
I would say instead of having a `const` component, make it a `function` component.
eg:
```js
function AboutTheCreators(){
   // ... the rest of the code 
}
export default AboutTheCreators;
```

Reply:
I think I fixed it.  Let me know if I did it correctly.

Reply:
You did 😄 , but you failed the `ESLint` test. You can run `npm run lint` it will show you the error. When you fix that error, The `ESLint` automation below gets fixed.

Reply:
I just fixed it with a new commit.  I forgot to take a semicolon out.